src/components/Coaching/AppealProgress/AppealProgress.tsx

```diff
@@ -44,16 +54,55 @@ export const AppealProgress = ({
             : goalText}
         </Typography>
         <Typography>
-          {currencyFormat(received, currency, locale)}(
-          {percentageFormat(received / goal, locale)})/
-          {currencyFormat(pledged, currency, locale)}(
-          {percentageFormat(pledged / goal, locale)})
+          <Tooltip
+            title={t('Received')}
+            placement="top"
+            arrow
+            PopperProps={{
```

Comment:
Since both of these `PopperProps` look identical, can we put them in a variable that's used in both places? You could also make a `TopTooltip` component if you wanted that just takes `title` and `children`. Might be overkill though, up to you.